coral/src/app/features/connectors/request/schemas/connector-request-form.ts

```diff
@@ -0,0 +1,40 @@
+import attempt from "lodash/attempt";
+import isError from "lodash/isError";
+import z from "zod";
+
+const connectorRequestFormSchema = z.object({
+  environment: z
+    .string()
+    .min(1, { message: "Selection Error: Please select an environment." }),
+  connectorName: z.string().min(5),
```

Comment:
I would add a custom message here. For TopicRequests, we have "Topic name can not be empty". Otherwise it says "String must contain at least 5 character(s)" which is a bit technical

Reply:
Excellent, i changed it to "Connector name can not be empty"